README.md

```diff
@@ -1,25 +1,46 @@
-# Percona Monitoring and Management
+# Percona Monitoring and Management 2.x
 
 ![PMM](https://www.percona.com/sites/default/files/pmm-logo.png)
 
-See the [PMM docs](https://www.percona.com/doc/percona-monitoring-and-management/index.html) for more information.
-
-## Main repositories
-
-* [pmm-server](https://github.com/percona/pmm-server)
-  * [qan-api](https://github.com/percona/qan-api)
-  * [qan-app](https://github.com/percona/qan-app)
-  * [grafana-dashboards](https://github.com/percona/grafana-dashboards)
-  * [percona-toolkit](https://github.com/percona/percona-toolkit)
-  * [pmm-manage](https://github.com/percona/pmm-manage)
-  * [pmm-managed](https://github.com/percona/pmm-managed)
-  * [pmm-update](https://github.com/percona/pmm-update)
-  * [pmm-server-packaging](https://github.com/percona/pmm-server-packaging)
-* [pmm-client](https://github.com/percona/pmm-client)
-  * [qan-agent](https://github.com/percona/qan-agent)
-  * [node_exporter](https://github.com/percona/node_exporter) - based on `github.com/prometheus/node_exporter`
-  * [mysqld_exporter](https://github.com/percona/mysqld_exporter) - based on `github.com/prometheus/mysqld_exporter`
-  * [mongodb_exporter](https://github.com/percona/mongodb_exporter) - based on `github.com/dcu/mongodb_exporter`
-  * [rds_exporter](https://github.com/percona/rds_exporter) - based on `github.com/Technofy/cloudwatch_exporter`
-  * [postgres_exporter](https://github.com/percona/postgres_exporter) - based on `github.com/wrouesnel/postgres_exporter`
-  * [proxysql_exporter](https://github.com/percona/proxysql_exporter)
+See the [PMM2 docs](https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html) for more information.
+
+## Repositories
+
+### PMM Server
+
+* [percona/pmm-managed](https://github.com/percona/pmm-managed/tree/PMM-2.0)
+* [percona/qan-api2](https://github.com/percona/qan-api2/tree/master)
+* [percona/qan-app](https://github.com/percona/qan-app/tree/PMM-2.0)
+* [percona/pmm-update](https://github.com/percona/pmm-update/tree/PMM-2.0)
+* [percona/percona-toolkit](https://github.com/percona/percona-toolkit/tree/3.0)
+* [percona/grafana-dashboards](https://github.com/percona/grafana-dashboards/tree/PMM-2.0)
+* [percona/pmm-manage](https://github.com/percona/pmm-manage/tree/PMM-2.0)
+* [percona/pid-watchdog](https://github.com/percona/pid-watchdog/tree/master)
+
+### PMM Client
+
+* [percona/pmm-agent](https://github.com/percona/pmm-agent/tree/master)
+* [percona/pmm-admin](https://github.com/percona/pmm-admin/tree/master)
+* [percona/node_exporter](https://github.com/percona/node_exporter)
```

Comment:
There you switched from links to branches to link to repositories. Why?

Reply:
Let's use branches only if this is not  a `default`